Jenkinsfile

```diff
@@ -466,7 +466,7 @@ pipeline {
             description: "")
         booleanParam(
             name: "TARGET_GFX908",
-            defaultValue: true,
+            defaultValue: env.BRANCH_NAME == env.NIGHTLY_BRANCH ? true : false,
```

Comment:
[Notice] AnyGPU stages ignore this. AnyGPU stages of non-nightly builds will occupy gfx908 nodes.

[R] Mention that this is a W/A for the MI100 node shortage problem.

Reply:
@atamazov yes, and it is left on purpose. We want all stages not to be restricted to gfx908, but sometimes anyGPU stages may occupy (but not limited by) gfx908 nodes and if issues found it is still legit.